### DISCUSSION TITLE
Stutters with new dev mod

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4707] [General]
Hi developers,

just for info, the following version produces a huge stutters while switching through the different EFB pages. Not seen when clicking on pages which are showing inop ... 

![grafik](https://user-images.githubusercontent.com/76531893/116928199-09b93200-ac5d-11eb-9ae7-6beffe632b43.png)

Greets Klaus

**Comment by ExampleWasTaken** [comment]
That will most probably be fixed with flypadOS-v2.

**Reply by CptBayer** [reply]
You are the best :-) !!!

Greets Klaus
